### PR TITLE
Added repo badges to readme

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # webidl.js
+[![Build Status](https://travis-ci.org/extensibleweb/webidl.js.svg?branch=master&maxAge=2592)](https://travis-ci.org/extensibleweb/webidl.js)
+[![NPM version](https://badge.fury.io/js/webidl.svg?maxAge=2592)](http://badge.fury.io/js/webidl)
```
Comment (marcoscaceres) [inline README.md]: What is fury.io?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+[![Dependency Status](https://img.shields.io/david/dev/extensibleweb/webidl.js.svg?maxAge=2592)](https://david-dm.org/extensibleweb/webidl.js)
```
Comment (marcoscaceres) [inline README.md]: Same here, what is "shields.io"?

Comment (amilajack) [inline README.md]: [shields.io](http://shields.io) is a free service that serves repo badges for GitHub, npm, ruby, and many other projects.

File: README.md (hunk 1 above)
Comment (amilajack) [inline README.md]: [fury.io](https://badge.fury.io) is a service that serves version badges

Comment (amilajack) [inline README.md]: [fury.io](https://badge.fury.io) is a service that serves version badges

**Comment by amilajack** [comment]
A preview of the readme can be [seen here](https://github.com/amilajack/webidl.js/blob/3453fdeddc42f9e79379c7cb6b6ed3c58da72d9f/README.md)

**Comment by marcoscaceres** [comment]
Ah, awesome... thanks.

**Comment by marcoscaceres** [comment]
Ah, look at that... devDeps are wayyyy out of date!

File: README.md (hunk 1 above)
Comment (marcoscaceres) [inline README.md]: That's great. Thanks for clarifying.

**Comment by amilajack** [comment]
Also the build status is showing as invalid because the travis builds aren't enabled yet. I can PR to update the deps

**Comment by marcoscaceres** [comment]
@amilajack, a PR to enable tests on travis would be amazing!

**Pull request merged**